src/calling/haplotypes.rs

```diff
@@ -53,36 +43,39 @@ impl Caller {
         // with each column after the first showing the fraction of the respective haplotype
         let mut posterior = m.event_posteriors();
         let mut wtr = csv::Writer::from_path(self.outcsv.as_ref().unwrap())?;
-        let mut haplotype_fractions = Vec::new();
+        wtr.write_record(&[
+            "density",
+            "odds",
+            "haplotype_a",
+            "haplotype_b",
+            "haplotype_c",
```

Comment:
Ok, and here one should not put hardcoded names, but instead build the haplotype columns from the input (i.e. the names coming from the kallisto data).